subprojects/common/junit-utils/src/main/java/com/avito/test/Flaky.kt

```diff
@@ -0,0 +1,9 @@
+package com.avito.test
+
+/**
+ * Unstable test marker.
+ * It's used for retry logic.
+ */
+annotation class Flaky(
```

Comment:
Isn't retention necessary here?

Reply:
Ok, I'll check it out in next PR

Reply:
`kotlin.annotation.AnnotationRetention`
As I think, the idea is to use the minimum required retention, to store only needed information.